JustSaying/AwsTools/MessageHandling/SqsPublisher.cs

```diff
@@ -53,7 +54,7 @@ public async Task PublishAsync(Message message, PublishMetadata metadata, Cancel
                     MessageResponseLogger.Invoke(responseData, message);
                 }
             }
-            catch (Exception ex)
+            catch (AmazonServiceException ex)
```

Comment:
Perhaps the scope of the try should be reduced to only the aws call here. If the logger stuff throws then this is a change in behaviour, perhaps that's fine but it could be misleading in it's current state.

Reply:
Good idea, done in 3 places